tests_thien/utilities/constant.py

```diff
@@ -0,0 +1,19 @@
+# DA
```

Comment:
What is #DA?
change it to meaning comment

Reply:
updated to Dashboard constant